### PR TITLE
Created wrapper for pkg

### PR DESCRIPTION
**Pull request by yggdr**
This should make it easier to use pkg with iocage jails without the need to
resort to resolve the jail id yourself.

### CODE REVIEW

File: lib/ioc-pkg [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+	local _name _command _jid
+	shift
+	_name="$1"
+	shift
+	_command="$*"
+
+	_jid=$(jls -h jid host.hostname | grep -w browser | cut -w -f 1)
+	if [ -z "$_jid" ]; then
+		echo "  ERROR: $_name not found."
+		exit 1
```
Comment (skarekrow) [inline lib/ioc-pkg:11]: Not needed, we have `__die` for this :)

File: lib/ioc-pkg [hunk 2]
```diff
@@ -0,0 +1,15 @@
+__pkg_wrapper () {
+	local _name _command _jid
+	shift
+	_name="$1"
+	shift
+	_command="$*"
+
+	_jid=$(jls -h jid host.hostname | grep -w browser | cut -w -f 1)
```
Comment (skarekrow) [inline lib/ioc-pkg]: browser?

**Comment by skarekrow** [comment]
Thanks for the PR! Just a couple notes before I will merge it :)

**Comment by yggdr** [comment]
Oops, fixed that "browser" thing

**Comment by skarekrow** [comment]
Thanks! Just replace the echo and exit 1 with die and rebase. Then I'll happily merge this.

Thanks again!

**Comment by yggdr** [comment]
I'm not sure I understand what you're referring to. I find no __die function declared anywhere. Replacing the exit with it yields a "not found" error.

**Comment by skarekrow** [comment]
```
 	if [ -z "$_jid" ]; then
 		echo "  ERROR: $_name not found."
 		exit 1
 	fi
```

Should become:

```
 	if [ -z "$_jid" ] ; then
 		__die "$_name not found."
 	fi
```

**Comment by skarekrow** [comment]
https://github.com/iocage/iocage/blob/develop/lib/ioc-common#L635 is where it's declared. I see now that you have opened this against the master branch. Please read the [Contributing Guidelines] (https://github.com/iocage/iocage/blob/master/CONTRIBUTING.md) before submitting a PR :)

**Comment by yggdr** [comment]
Oh wow, brainfarts upon brainfart these last few days... Will resubmit a cleaned-up PR against develop.

**Comment by skarekrow** [comment]
Hehe it happens to all of us! I appreciate it :)

**Pull request closed without merging**